configuration/configuration.go

```diff
@@ -80,7 +80,7 @@ func NewGenericConfiguration() GenericConfiguration {
 // NewPulsarConfiguration creates new default configuration for pulsar
 func NewPulsarConfiguration() PulsarConfiguration {
 	return PulsarConfiguration{
-		Log:      NewLog(),
```

Comment:
don't break formatting please (or at least keep it consistent)

Reply:
ok